### DISCUSSION TITLE
为什么选择多P但却只有一个被选中

### DISCUSSION THREAD

**Discussion by xiongaox** [the1812/Bilibili-Evolved#3804] [提问 / Q&A] [answered]
如图
<img src="https://file.moetu.org/images/2022/11/05/Snipaste_2022-11-05_21-04-57521989e449c36061.png"></img>

**Comment by Ethkuil** [comment, marked as answer]
因为合集中的每个视频是独立的，不是一个视频的分P（其中一个体现为：它们不共享评论区）。

**Comment by Geeyun-JY3** [comment]
如果想要下载合集所有视频可以留意这里：#3240
